README.md

```diff
@@ -95,3 +95,32 @@ cobra-cli add <new-command>
 | `dev clean all`            | Deletes the local data, downloaded binaries, and config files for the Astria stack. |
 | `sequencer create-account` | Generate an account for the Sequencer.                                              |
 | `sequencer get-balance`    | Get the balance of an account on the Sequencer.                                     |
+
+### Instances
+
+The `dev init`, `dev run`, and `dev clean` commands above all have an optional
```

Comment:
```suggestion
The `dev init`, `dev run`, and `dev clean` commands all have an optional
```

Reply:
I try to refrain from referencing locations like this without a link because things will get moved around but they won't think to update other sections. I'd just remove the location reference.